release.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cargo release --workspace `comm -23 <(cargo ws ls --json | jq -r '.[].name' | sort) <(cargo ws changed --json | jq -r '.[].name' | sort) | xargs -I {} echo --exclude {}` --no-publish --no-push $2 $1 
```

Comment:
Hmm, this one is quite a bit of code golf and no help to interpret it.

First of all, it took me a while to realize I needed `cargo-workspaces`, not `cargo-workspace` or `cargo-ws` for this.

Second, it seemed that this was close to just doing `cargo release -p <pkgid>` for the output of `cargo changed`, with one relevant difference: `luwen` would be included in that case.

Either way, this is the kind of little utility script that would be useful to have a few comments to your friendly coworkers when they have to debug weird behavior. It wouldn't hurt to bounce the output from the two commands through variables either, to make it easier to debug in steps (or via `bash -x`) if needed.